client/layout/global-sidebar/style.scss

```diff
@@ -1,12 +1,16 @@
+@import "@wordpress/base-styles/mixins";
 @import "@wordpress/base-styles/variables";
 @import "@automattic/typography/styles/variables";
 $brand-text: "SF Pro Text", $sans;
 .global-sidebar {
 	bottom: 0;
+	contain: content;
 	display: flex;
 	flex-direction: column;
 	left: 0;
 	height: 100%;
+	overflow-y: auto;
```

Comment:
It makes the sidebar scrollable on horizontally

![image](https://github.com/Automattic/wp-calypso/assets/13596067/1a381009-bed5-42c3-b0e0-262ef8ddc054)